client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -11,41 +11,42 @@ import GridiconMoney from 'gridicons/dist/money';
  */
 import edit from './edit';
 import save from './save';
-import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { __, _x } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+/**
+ * Styles
+ */
+import './editor.scss';
 
 registerBlockType( 'jetpack/simple-payments', {
 	title: __( 'Payment button' ),
 
 	description: __(
-		'Simple Payments lets you create and embed credit and debit card payment buttons on your WordPress.com and Jetpack-enabled sites with minimal setup.'
+		'Simple Payments lets you create and embed credit and ' +
+			'debit card payment buttons on your WordPress.com and ' +
+			'Jetpack-enabled sites with minimal setup.'
 	),
 
 	icon: <GridiconMoney />,
 
 	category: 'jetpack',
 
-	keywords: [ __( 'simple payments' ), 'PayPal' ],
+	keywords: [ __( 'Simple Payments' ), _x( 'shop', 'block search term' ), 'PayPal' ],
 
 	attributes: {
 		currency: {
 			type: 'string',
-			default: 'USD',
 		},
-		description: {
+		content: {
 			type: 'string',
 			default: '',
 		},
 		email: {
 			type: 'string',
 			default: '',
 		},
-		formattedPrice: {
-			type: 'string',
-			default: '',
-		},
 		multiple: {
 			type: 'number',
```

Comment:
This is really a boolean value, isn't it? We can handle transforming the API response.

Reply:
WordPress and boolean values are confusing since they're moved around as `0` and `1`. I'd treat it as boolean as much as possible but I'm afraid from/to API we need to send it as an integer.

Reply:
Related: https://github.com/Automattic/jetpack/pull/10402#discussion_r230837551

I'd like to isolate API concerns from component logic. Indicating this is a number is confusing, I'm happy to add `toApi`/`fromApi` as necessary.